lib/LogoGC/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import logoSVG from "../../assets/logo.svg";
+
+/**
+ * @usage
+ *
+ * <Logo url="http://example.com/myimage.png" alt="Alt Tag" />
+ */
+
+export function LogoGC(props: any) {
+  const LogoPath = props.url || logoSVG;
+  const image =
+    typeof LogoPath === "string" ? (
+      <img src={props.url} alt={props.alt} className="logo__image" />
+    ) : (
+      <span className="logo__image">
+        <LogoPath />
```

Comment:
This points to the same svg as the new bynder one, we need the old svg in place :)

Reply:
Added old SVG back and also added a new story so it's clear what's what.